modules/web-discovery-project/sources/web-discovery-project.es

```diff
@@ -3228,7 +3248,7 @@ const WebDiscoveryProject = {
         if (cstr.length > WebDiscoveryProject.rel_segment_len) {
           if (isHash(cstr)) return true;
 
-          if (isHashProb(cstr.toLowerCase())) {
+          if (isHash(cstr.toLowerCase(), { threshold: 0.0225 })) {
```

Comment:
why not use the default threshold?

Reply:
It is quite an old piece of code which I've just updated to use the new `hash-detector` module (haven't changed the threshold here), but I think that the logic here is that we first check the string itself and then the lowercased version. But since we rely on a case of a character while checking for a hash we increase the threshold for a lowercased version of the string to avoid a lot of false positives